No check.mcode target

The code suggests that ```check``` is the target to test the code, but it fails:
```
gmake[1]: *** No rule to make target 'check.mcode', needed by 'check'.  Stop.
```
also
```
gmake[1]: *** No rule to make target 'check.llvm', needed by 'check'.  Stop.
```

--- a/Makefile.in
+++ b/Makefile.in
@@ -132,8 +132,6 @@ install: install.$(backend)
 
 uninstall: uninstall.$(backend)
 
-check: check.$(backend)
-
 copy-sources: copy-sources.$(backend)
 
 libs: libs.vhdl.$(backend)

